server/bin/fxa-content-server.js

```diff
@@ -87,7 +87,11 @@ function makeApp() {
     app.use(csp({ rules: cspRulesBlocking }));
   }
   if (config.get('csp.reportOnlyEnabled')) {
-    app.use(csp({ rules: cspRulesReportOnly }));
+    // There has to be more than a `reportUri`
+    // to enable reportOnly CSP.
+    if (Object.keys(cspRulesReportOnly.directives).length > 1) {
```

Comment:
I was of 2 minds of whether to even keep the report-only module and middleware. Since we have no reportOnly rules and are not adding the middleware, a case can be made for YAGNI and that the entire module/config items should be removed until we need them again.

Though I can also make the case that someone is going to tell us we need to be more strict with our rules, and we'll want reportOnly for then and we'll have to dig out the infrastructure. Of course, if anybody still remembers it was there, we could dig it out of source control when needed.